app/src/main/java/org/systers/mentorship/utils/CommonUtils.kt

```diff
@@ -20,7 +20,7 @@ object CommonUtils {
      */
     fun getErrorResponse(@NonNull throwable: Throwable): CustomResponse {
         val httpException = throwable as HttpException
-        val response = httpException.response().errorBody()?.string()
+        val response = httpException.response()?.errorBody()?.string()
         return gson.fromJson(response.toString(), CustomResponse::class.java)
     }
 }
```

Comment:
Response here can be null

Reply:
It can be null (type of `response` is `String?`)

BTW That's where Kotlin's implicitness becomes a little bit cumbersome:)

Reply:
So I recommend
```
response?.let{return gson.fromJson(it, CustomResponse::class.java)}
// if response is null then return new message 
```
 hope I didn't made any mistake I'm on mobile :)

Reply:
Fixed according to your suggestion (no mistake ;) )